include/swift/AST/ASTContext.h

```diff
@@ -372,13 +372,13 @@ class ASTContext final {
   /// Cache of module names that fail the 'canImport' test in this context.
   mutable llvm::StringSet<> FailedModuleImportNames;
 
-  /// Set if a `-module-alias` was passed. Used to store mapping between module aliases and
-  /// their corresponding real names, and vice versa for a reverse lookup, which is needed to check
-  /// if the module names appearing in source files are aliases or real names.
-  /// \see ASTContext::getRealModuleName.
+  /// Set if a `-module-alias` was passed. Used to store mapping between module syntax names and
+  /// their corresponding binary names, and vice versa for a reverse lookup, which is needed to check
+  /// if the module names appearing in source files are syntax names or binary names.
+  /// \see ASTContext::getBinaryModuleName.
   ///
-  /// The boolean in the value indicates whether or not the entry is keyed by an alias vs real name,
-  /// i.e. true if the entry is [key: alias_name, value: (real_name, true)].
+  /// The boolean in the value indicates whether or not the entry is keyed by a syntax name vs binary name,
+  /// i.e. true if the entry is [key: syntax_name, value: (binary_name, true)].
   mutable llvm::DenseMap<Identifier, std::pair<Identifier, bool>> ModuleAliasMap;
```

Comment:
A side note: Collections that have the same type on both sides with only a semantic difference between them make me nervous. This isn't something to do now, but it might be worth introducing separate `Identifier` wrapper types like `Module::BinaryName` and `Module::SyntaxName` (to take your spelling), delete their mutual conversion operators, and see if that helps clarify things here.

Reply:
That's a good idea.